### PR TITLE
Additional Front End Developer 1 Competencies

### PR DESCRIPTION
**Pull request by korgon**
Adding new competencies used in the Front End Developer 1 role:
* browser APIs
* HTML
* CSS
* ui design patterns

### CODE REVIEW

File: competencies/browser-apis.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Competency - Browser APIs
+
+When writing code for the Web, there are a large number of Browser APIs available for use. These APIs are constantly being created and improved as new technologies emerge and are incorporated into the web experience. Having a solid understanding of the available APIs is important for knowing what can be done, and how to do it faster and easier.
+
+## How do you prove it?
+
+You can explain how you've used some of the more commonly used APIs.
```
Comment (codeallthethingz) [inline competencies/browser-apis.md:7]: give it a number.  "some" is hard to validate.

File: competencies/browser-apis.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+* [Web Speech API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Speech_API)
+* [Vibration API](https://developer.mozilla.org/en-US/docs/Web/API/Vibration_API)
+* [Pointer Lock API](https://developer.mozilla.org/en-US/docs/Web/API/Pointer_Lock_API)
+* [WebSocket API](https://developer.mozilla.org/en-US/docs/Web/API/WebSockets_API)
+
+See a [full list of browser APIs](https://developer.mozilla.org/en-US/docs/Web/API).
+
+## How do you improve it?
+
+Experiment with some of the more interesting APIs and build something awesome.
```
Comment (codeallthethingz) [inline competencies/browser-apis.md:38]: whats that site doesitwork.com for browser compatibility?

File: competencies/browser-apis.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+# Competency - Browser APIs
+
+When writing code for the Web, there are a large number of Browser APIs available for use. These APIs are constantly being created and improved as new technologies emerge and are incorporated into the web experience. Having a solid understanding of the available APIs is important for knowing what can be done, and how to do it faster and easier.
+
+## How do you prove it?
+
+You can explain how you've used some of the more commonly used APIs.
+
+## Commonly used APIs
```
Comment (codeallthethingz) [inline competencies/browser-apis.md:9]: should be a three level heading if it's in the prove it

File: competencies/css.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+* You can explain what display 'flex' and 'grid' are, and what the benefits of using both are.
+* You can explain what a media query is and when it is used.
+
+## How do you improve it?
+
+[Animation](https://developer.mozilla.org/en-US/docs/Web/CSS/animation)  
+[Color Contrast](https://www.w3schools.com/accessibility/accessibility_color_contrast.php)  
+[CSS Variables](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)  
+
+## Related technologies
```
Comment (codeallthethingz) [inline competencies/css.md:26]: should be a level 3

File: competencies/html.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+* You can describe the various elements and attributes that make up a functioning form.
+* You know about the various multimedia elements that are available.
+* You know what `noscript` and `script` tags are, what they are used for and when you might use a linked script, in-document script and an inline-script.
+
+## How do you improve it?
+
+[Learn HTML](https://html.com/)  
+[Learn Accessibility](https://developer.mozilla.org/en-US/docs/Learn/Accessibility/HTML)  
+[Learn Forms](https://developer.mozilla.org/en-US/docs/Learn/Forms)  
+[Tag Reference](https://developer.mozilla.org/en-US/docs/Web/HTML/Element)
```
Comment (codeallthethingz) [inline competencies/html.md:22]: maybe add semantic html something or other

File: competencies/ui-design-patterns.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
+# Competency - UI Design Patterns
+
+Patterns are reusable solutions to commonly seen problems. UI design patterns consists of structural and behavioral features that are familiar to users. When designing UI for the Web, there are a large number of UI design patterns that exist that can be leveraged.
+
+## How do you prove it?
+
+* Talk about 5 different UI design patterns and when they are used.
+
+## Common Web UI design patterns
```
Comment (codeallthethingz) [inline competencies/ui-design-patterns.md:9]: should be 3 heading under the improve it section

File: competencies/ui-design-patterns.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
+# Competency - UI Design Patterns
+
+Patterns are reusable solutions to commonly seen problems. UI design patterns consists of structural and behavioral features that are familiar to users. When designing UI for the Web, there are a large number of UI design patterns that exist that can be leveraged.
+
+## How do you prove it?
+
+* Talk about 5 different UI design patterns and when they are used.
```
Comment (codeallthethingz) [inline competencies/ui-design-patterns.md:7]: Couple more things.  Maybe build a thing using one of the patterns

**Review by codeallthethingz — COMMENTED** [review]
awesome work.  Couple minor tweaks.

**Comment by korgon** [comment]
Minor tweaks made. Awaiting re-review.

**Review by codeallthethingz — APPROVED** [review]
excellent

**Pull request merged**